lib/rules/no-debug.ts

```diff
@@ -32,11 +33,25 @@ export default createTestingLibraryRule<Options, MessageIds>({
     messages: {
       noDebug: 'Unexpected debug statement',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          utilNames: {
+            type: 'array',
+            items: {
+              type: 'string',
+              enum: DEBUG_UTILS,
+            },
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
   },
-  defaultOptions: [],
+  defaultOptions: [{ utilNames: ['debug', 'logTestingPlaygroundURL'] }],
```

Comment:
Use `DEBUG_UTILS` by default

```suggestion
  defaultOptions: [{ utilNames: [...DEBUG_UTILS] }],
```

Reply:
The issue with this is that any change to `DEBUG_UTILS` will implicitly affect this rule, which semver thoughts aside, is a very easy way to introduce an unintentional change.

If you're happy with that risk I don't mind changing this.

Reply:
Let's keep the array as it was